docs/DeveloperGuide.md

```diff
@@ -280,40 +280,47 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * `   | teaching assistant | edit a student's information                    | update any changes to a student's information quickly                                    |
+| `* * *`  | teaching assistant | delete a student from my classes                | remove students who decide to drop the module                                            |
+| `* * `   | teaching assistant | clear all students in my classes                | clear all students at once after a semester                                              |
+| `* * *`  | teaching assistant | mark students' attendance                       | see cumulative class attendance grade at the end of the semester                         |
+| `* * *`  | teaching assistant | mark students' participation                    | see the cumulative class participation grade at the end of the semester                  |
+| `* * *`  | teaching assistant | see a pie chart summarizing my students' grades | keep track of my students' performances                                                  |
+| `* *`    | teaching assistant | extract students' emails                        | mass email a group of students                                                           |
+| `* * *`  | teaching assistant | find a student by their name                    | get a student information quickly without having to go through the entire list           |
+| `* * *`  | teaching assistant | find a student by their student ID              | get their name quickly while grading their assignments without having to go through the entire list |
+| `* * *`  | teaching assistant | find a student by their tutorial                | locate details of all students in the same tutorial without having to go through the entire list |
+| `* * *`  | teaching assistant | find a student by their module                  | locate details of of all students in the same module without having to go through the entire list |
+| `* * *`  | teaching assistant | add a new tutorial slot                         | track the list of tutorials that I have                                                  |
+| `* * `   | teaching assistant | edit a tutorial slot                            | update the details of an existing tutorial quickly                                       |
+| `* * *`  | teaching assistant | delete a tutorial slot                          | remove unwanted tutorials                                                                |
+| `* *`    | teaching assistant | clear all tutorial slots                        | clear all tutorial at once after a semester                                              |
+| `* * *`  | teaching assistant | add a new consultation slot                     | track the list of consultations that I have                                              |
```

Comment:
I think you should update the example for adding a new consultation since the time range in the current message is in wrong format.